genesyscloud/journey_views/resource_genesyscloud_journey_views.go

```diff
@@ -57,12 +57,14 @@ func createJourneyView(ctx context.Context, d *schema.ResourceData, meta interfa
 
 func updateJourneyView(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	name := d.Get("name").(string)
+	versionId := d.Get("version").(int)
```

Comment:
It is possible that if the user does not specify a version, casting to int here *could* cause a conversion error to be thrown. I would change this slightly to make sure this does not happen 

```diff
-versionId := d.Get("version").(int)
+versionId, _ := d.Get("version").(int)
```

Reply:
No risk because the version is coming from API. It is computed information. So not provided by user and API returns always a version